### PR TITLE
Add settings to allow schema registrys to share a ZK.

### PR DESCRIPTION
**Pull request by ddryden**
https://office.brandwatch.com/jira/browse/DI-1048

### CODE REVIEW

File: templates/schema-registry.properties.erb [hunk 1]
```diff
@@ -17,6 +17,16 @@ port=<%= @port %>
 debug=<%= @debug %>
 
 kafkastore.connection.url=<%= @kafkastore_connection_url %>
+<% if @zk_namespace -%>
+schema.registry.zk.namespace=<%= @zk_namespace %>
+<% end -%>
+<% if @group_id -%>
```
Comment (richardbadman) [inline templates/schema-registry.properties.erb:23]: I know obvs. that `undef` means `undefined`, but would this ever resolve if it's undefined ?

**Review by richardbadman — COMMENTED** [review]
Just a quick Q with the `.erb` file , but it looks good to me

File: templates/schema-registry.properties.erb (hunk 1 above)
Comment (ddryden) [reply templates/schema-registry.properties.erb:23]: Good question, its my understanding that it won't but ERB is ruby and the undef gets translated into nil according to this doc:
https://puppet.com/docs/puppet/5.3/lang_template_erb.html#manipulating-data

**Review by richardbadman — APPROVED** [review]

File: templates/schema-registry.properties.erb (hunk 1 above)
Comment (ddryden) [reply templates/schema-registry.properties.erb:23]: Think its good:
```
irb(main):010:0> def test(a)
irb(main):011:1>   if a
irb(main):012:2>     puts 'meh'
irb(main):013:2>   else
irb(main):014:2*     puts 'yay'
irb(main):015:2>   end
irb(main):016:1> end
=> :test
irb(main):017:0> thing = nil
=> nil
irb(main):018:0> test(thing)
yay
=> nil
irb(main):019:0> thing = 'lksdhfeu'
=> "lksdhfeu"
irb(main):020:0> test(thing)
meh
=> nil

```

**Pull request merged**